cmd/onedriver-launcher/main.go

```diff
@@ -85,10 +85,15 @@ func activateCallback(app *gtk.Application) {
 	mountpointBtn.SetTooltipText("Add a new OneDrive account.")
 	mountpointBtn.Connect("clicked", func(button *gtk.Button) {
 		mount := ui.DirChooser("Select a mountpoint")
+		if mount == "" {
```

Comment:
My only quibble with this PR is that this "intercepts" the log message from being printed on the CLI if the user cancels the operation (I can fix this in another commit).

Reply:
Ah yes, I was a bit unsure whether cancelling the dialog was considered a log-worthy event. Thank you for clarifying :+1: